src/renderer/widgets/PasswordInputField.hpp

```diff
@@ -39,6 +39,7 @@ class CPasswordInputField : public IWidget {
     Vector2D    configSize;
 
     std::string halign, valign, configFailText, outputStringPort, configPlaceholderText;
+    u_int64_t   configFailTimeoutMs;
```

Comment:
`uint64_t`

also please initialize it.

Reply:
Typo, sorry.
Is `uint64_t    configFailTimeoutMs = 0;` correct, or it should be 2000? I though the following line was enough:

```
configFailTimeoutMs      = std::any_cast<Hyprlang::INT>(props.at("fail_timeout"));
```

There is `fadeTimeoutMs` at the end of the class which is not initialized either, should that be fixed too?